README.md

```diff
@@ -1281,6 +1284,16 @@ defaults write com.apple.PowerChime ChimeOnAllHardware -bool false && \
 killall PowerChime
 ```
 
+From macOS 10.13 (High Sierra):
```

Comment:
This needs to read `From macOS 10.13 (High Sierra) on` to match the style of existing entries.